dapr/proto/runtime/v1/dapr.proto

```diff
@@ -33,7 +33,7 @@ service Dapr {
   rpc PublishEvent(PublishEventRequest) returns (google.protobuf.Empty) {}
 
   // Invokes binding data to specific output bindings
-  rpc InvokeBinding(InvokeBindingRequest) returns (google.protobuf.Empty) {}
+  rpc InvokeBinding(InvokeBindingRequest) returns (InvokeBindingResponse) {}

```

Comment:
Should we expose query capability for operations from Bindings?
https://github.com/dapr/components-contrib/blob/master/bindings/output_binding.go#L12

Reply:
I tend to say no, unless we come up with a concrete use case for this.

Reply:
Reason I proposed it because currently user has to look at the code of component to see what operations it supports. Currently components don't check the type of operations they support on Invoke so they can't give a meaningful error with list of supported operations.

Reply:
The docs will be updated to show which operations are supported. The dapr runtime checks the supported operations and returns an error if its not supported:

https://github.com/dapr/dapr/pull/1608/files#diff-a6a78e03551b53fec882ef23c9762278R505